pkg/cosign/tuf/client.go

```diff
@@ -97,11 +95,11 @@ func (b *ByteDestination) Delete() error {
 
 // Retrieves a local target, either from the cached root or the embedded metadata.
 func getLocalTarget(name string) (fs.File, error) {
-	if _, err := os.Stat(CosignCachedTargets()); !os.IsNotExist(err) {
+	if f, err := os.Open(filepath.Join(CosignCachedTargets(), name)); err == nil {
 		// Return local cached target
-		return os.Open(path.Join(CosignCachedTargets(), name))
+		return f, nil
 	}
-	return root.Open(path.Join("repository", "targets", name))
+	return root.Open("repository/targets/" + name)
```

Comment:
why doesn't this need `filepath.Join`? 🤔

Reply:
the `embed.FS` it creates uses unix-like path separators, even on Windows /shrug

Reply:
Woof, can you add comments to that effect?

Reply:
done (at the helper fn)